### ISSUE TITLE
ARM Templates do not allow specification of deployment location

### ISSUE THREAD

**Issue by jberkers42** [F5Networks/f5-azure-arm-templates#218]
## Do you already have an issue opened with F5 support?  No
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

## Description
While deploying an F5 Resource in to Azure, we would like to use a single resource-group within our Subscription, and deploy separate resources to different regions.  Currently the F5 resources are deployed to the same region as the Resource Group.

So far we have worked around this by modifying the template to include a location parameter, and adjusting the location variable to use the supplied parameter value instead of using the location from the Resource Group.

We included the following parameter stanza:

```
         "location": {
             "metadata": {
                 "description": "The location to which you want to deploy the BIG-IP VEs."
             },
             "type": "string"
         },
```

We updated the location variable from:

```
"location": "[resourceGroup().location]",
```

to

```
"location": "[parameters('location')]",
```

## Template
We have encountered this issue with the following templates:

supported/standalone/3nic/existing-stack/byol/azuredeploy.json
supported/failover/same-net/via-api/n-nic/existing-stack/byol/azuredeploy.json

We have encountered this issue since template versions 6.x to 9.3.0.0

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: 5

Severity level definitions:
 1. **Severity 1 (Critical)** : Defect is causing systems to be offline and/or nonfunctional. immediate attention is required.
 2. **Severity 2 (High)** : Defect is causing major obstruction of system operations.
 3. **Severity 3 (Medium)** : Defect is causing intermittent errors in system operations.
 4. **Severity 4 (Low)** : Defect is causing infrequent interuptions in system operations.
 5. **Severity 5 (Trival)** : Defect is not causing any interuptions to system operations, but none-the-less is a bug.

**Comment by shyawnkarim** [comment]
Thanks for reporting this issue. We are now tracking this enhancement request internally with ID ESECLDTPLT-2443.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-azure-arm-templates-v2).
